runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -662,7 +663,7 @@ public void sendRequest(CorfuMessage.RequestPayloadMsg payload, long epoch,
 
         // Write this message out on the channel
         channel.writeAndFlush(request, channel.voidPromise());
-        log.trace("Sent one-way request message: {}", request.getHeader());
+        log.trace("Sent one-way request message: {}", TextFormat.shortDebugString(request.getHeader()));
     }
 
     /**
```

Comment:
There's also an instance in `validateClientId`.

Reply:
Added